.simplecov

```diff
@@ -0,0 +1,14 @@
+SimpleCov.profiles.define 'pivorak_cov' do
+  add_filter '/spec/'
+  add_filter '/config/'
+
```

Comment:
@denysmedynskyi helpers might be OK now as we **dryed** a lot of templates code there